### DISCUSSION TITLE
Native integration with Qwik

### DISCUSSION THREAD

**Discussion by aabmets** [mantinedev/mantine#2599] [Feature requests]
While the Qwik framework features a bridge for React libraries, using it 
does not leverage the whole power of the Qwik framework. Since Mantine is 
somewhat of a heavy library, it would benefit from any kind of optimizations 
that could be implemented for it. If Mantine could have native support for Qwik, 
web clients would experience significant time-to-interactive improvements, when
the Mantine library is being used for a web app.

**Comment by fungilation** [comment]
Further thoughts now that Qwik 1.0 and Qwik City looks ready for production use?
